### DISCUSSION TITLE
Close channel problem

### DISCUSSION THREAD

**Discussion by amtexugo** [cryptosharks131/lndg#320] [General]
Hello friends!

I'm getting in touch with you guys because of a channel opened with you that has been closing for two weeks but has not completed closing. Could you let me know what the procedure is? If something is wrong? Here's the id and the print... 

![Capturar_2023_08_23_13_27_28_80](https://github.com/cryptosharks131/lndg/assets/127704297/e207ce3a-0f1b-48f1-ac38-75e876c07030)
![Capturar_2023_08_23_13_27_38_805](https://github.com/cryptosharks131/lndg/assets/127704297/c972c335-ee56-4908-8f13-582ad8b0d607)

channel id: 875883057335566337

Thanks

**Comment by cryptosharks131** [comment]
Looks like your closing tx is missing from the LND response. Try giving LND a restart and see if the closing tx populates!
